### DISCUSSION TITLE
About transforms.SaveImaged, how to change the file name when saving image

### DISCUSSION THREAD

**Discussion by uto-lt** [Project-MONAI/MONAI#6675] [Q&A]
Hello,
The scenario is that I have an image named `xxxx_0000.nii.gz`, and I want to save the segmentation of the image as `xxxx.nii.gz`. Just as you can see, I want to change the file name when saving the segmentation of an image.

Maybe I don't know enough about the current framework, I don't know whether the current `SaveImaged` can do this. Could you please tell me how to make it with the current framework?

I added some code locally to achieve this, if the current framework can do it, I am pleased to make a PR and make some contribution.

Looking forward to your reply!

**Comment by KumoLiu** [comment]
Hi @uto-lt, I think your question is quite similar to this [issue](https://github.com/Project-MONAI/MONAI/discussions/6188). You can define your own `FolderLayout`.
https://github.com/Project-MONAI/MONAI/blob/e7fb74f32b5371bb54bff7a47447df31d06d8edf/monai/transforms/io/array.py#L377

Hope it can help you, thanks!
